cocos/core/builtin/builtin-res-mgr.jsb.ts

```diff
@@ -37,11 +40,8 @@ const ImageAsset = jsb.ImageAsset;
 const BuiltinResMgr = jsb.BuiltinResMgr;
 const builtinResMgrProto = BuiltinResMgr.prototype;
 
-const oldInitBuiltinRes = builtinResMgrProto.initBuiltinRes;
-
-builtinResMgrProto.initBuiltinRes = function (device: Device): Promise<void> {
+builtinResMgrProto.init = function (): Promise<void> {
```

Comment:
Will BuiltinResMgr be used for non-builtin resources?
If it could be, `Builtin` prefix seems not to be necessary.

Reply:
BuiltinResMgr is still used to fetch builtin resources. all assets can be fetched in assetManager